### PR TITLE
First steps for getting files from the grid. 

### PR DESCRIPTION
**Pull request by kdungs**
Needs input concerning filenames.

### CODE REVIEW

**Comment by betatim** [comment]
@kdungs _bump_

**Comment by kdungs** [comment]
Should be good now. Please review and merge if okay.

**Comment by betatim** [comment]
Can you rebase this so it merges?

File: 05-files-from-grid.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+>
+> * Obtain a {,x,µ}dst file from the grid
+
+> ## Prerequisites {.prereq}
+>
+> * Access to lxplus
+> * A working grid certificate
+> * Completed section 3
+
+In section ["Finding a Dataset in the Bookkeeping"](03-.html), we obtained a file called `MC_2012_27163003_Beam4000GeV2012MagDownNu2.5Pythia8_Sim08e_Digi13_Trig0x409f0045_Reco14a_Stripping20NoPrescalingFlagged_ALLSTREAMS.DST.py` which contains the following section:
```
Comment (betatim) [inline 05-files-from-grid.md]: make it 03-bookkeeping.html

**Comment by kdungs** [comment]
This is a fast-forward now. Can be merged via the GitHub UI now.

**Comment by betatim** [comment]
You created a commit in the history though that fixes the merge conflict.
What steps did you do?

**Comment by kdungs** [comment]
```
git rebase --onto tim/gh-pages gh-pages
git pull
```

**Comment by kdungs** [comment]
Probably did it wrong. We used to merge locally.

**Comment by betatim** [comment]
We should make a practice repository and have a play.

**Pull request merged**